org.adempiere.ui/src/org/compiere/grid/CreateFromInvoice.java

```diff
@@ -547,6 +547,14 @@ else if (M_RMALine_ID != 0)
 				if (inoutLine != null)
 				{
 					invoiceLine.setShipLine(inoutLine);		//	overwrites
+					if(invoiceLine.getC_UOM_ID()!=inoutLine.getC_UOM_ID()) {
+						invoiceLine.setC_UOM_ID(inoutLine.getC_UOM_ID());						
+						BigDecimal PriceEntered = MUOMConversion.convertProductFrom (invoice.getCtx(), M_Product_ID, 
+								inoutLine.getC_UOM_ID(), invoiceLine.getPriceEntered());
+							if (PriceEntered == null)
+								PriceEntered = invoiceLine.getPriceEntered(); 						

```

Comment:
this isn't a safe fallback, better to raise error here if conversion is not possible.

Reply:
code updated for throw error when no conversion found